README.md

```diff
@@ -83,17 +83,18 @@ supported versions of Kubernetes clusters.
 
 #### Compatibility matrix
 
```

Comment:
Do we indicate that we drop v6 support here or in the actual release?

Reply:
We support 3 stable releases, so we will drop v6 support after we release v9 GA version.